apps/trading/e2e/conftest.py

```diff
@@ -58,7 +58,12 @@ def do_GET(self):
 
 # Start VegaServiceNull
 @contextmanager
-def init_vega(request=None):
+def init_vega(request=None, local_server: bool = False):
```

Comment:
```suggestion
def init_vega(request=None):
```

**thought (blocking)**:  It feels to me like this could be inverted and the whole diff could be simplified. If the environment variable was not set, it could just use a default of 8001? The 5 lines below seem like they cover the logic, much more simply

Reply:
I think I've come up with a good solution with using the .env file. Please let me know what you think